core/src/components/back-button/back-button.ios.scss

```diff
@@ -13,10 +13,21 @@
   --color: #{$back-button-ios-color};
   --icon-margin-end: 1px;
   --icon-margin-start: -4px;
-  --icon-font-size: 1.6em;
+  /**
+   * This value used to be 1.6em which would evaluate
+   * to 1.6em * 17px (the font size of the button) = 27.2px.
+   * As a result, we set that to the minimum font size.
+   */
+  --icon-font-size: clamp(27.2px, 1.6rem, 36px);
```

Comment:
Do we have a tech debt ticket to fix these weird font sizes?

Reply:
Brandy and I discussed offline and we kept 1.6em because the icon should be sized relative to the text, and the text already has a) dynamic type support and b) font size clamping